------------------------------------------------------------
revno: 735
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Wed 2012-11-14 14:28:18 -0500
message:
  set shell for default ubuntu user to /bin/bash
=== modified file 'config/cloud.cfg'
--- old/config/cloud.cfg	2012-11-13 19:18:26 +0000
+++ new/config/cloud.cfg	2012-11-14 19:28:18 +0000
@@ -82,6 +82,7 @@
      lock_passwd: True
      gecos: Ubuntu
      groups: [adm, audio, cdrom, dialout, floppy, video, plugdev, dip, netdev]
+     shell: /bin/bash
    # Other config here will be given to the distro class and/or path classes
    paths:
       cloud_dir: /var/lib/cloud/

